operator/src/main/java/org/bf2/operator/operands/KafkaCluster.java

```diff
@@ -475,6 +475,12 @@ private KafkaClusterTemplate buildKafkaTemplate(ManagedKafka managedKafka, int r
 
         podTemplateBuilder.addAllToTolerations(OperandUtils.profileTolerations(managedKafka));
 
+        if (replicas == 1) {
+            podTemplateBuilder.editOrNewMetadata()
+                    .addToAnnotations("cluster-autoscaler.kubernetes.io/safe-to-evict", "true")
```

Comment:
should the annotation be added to other pods as well of a developer instance:
- zookeeper 
- canary
- kafka-admin
- kafka-exporter
?

Reply:
The impetus here is the change to strimzi pod sets - the other pods will be managed by a deployment, which is fine for the auto-scaler to move.